tests/unit/payment-methods/test-class-upe-payment-gateway.php

```diff
@@ -1888,19 +1888,6 @@ function( $argument ) {
 		}
 	}
 
-	/**
-	 * @dataProvider maybe_filter_gateway_title_data_provider
-	 */
-	public function test_maybe_filter_gateway_title_with_no_additional_feature_flags_enabled( $data ) {
```

Comment:
[Filtering occurs for non-deferred UPE only](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/payment-methods/class-upe-payment-gateway.php#L1152), so this test is not needed anymore as it will fail because of UPE enabled by default.